nilearn/signal.py

```diff
@@ -20,8 +20,9 @@
 NP_VERSION = distutils.version.LooseVersion(np.version.short_version).version
 
 
-def _standardize(signals, detrend=False, normalize=True):
-    """ Center and norm a given signal (time is along first axis)
+def _standardize(signals, detrend=False, standardize=True, 
+                 standardize_strategy='energy'):
```

Comment:
old design normalize=True is same as standardize_strategy='zscore' in our new design ?